### PR TITLE
Fix login via facebook

### PR DESCRIPTION
**Pull request by ghost**
Use of the new Facebook login SDK instead of the old one.

### CODE REVIEW

**Review by emiltin — COMMENTED** [review]
hi, i'm going to merge this, but will clean up the commit history a bit first. next time, please avoid revert og merge messages in a PR, and avoid adding large amounts of tiny commits.
the goal is that the commit history is easy to read, and consists of meaningful blocks of changes.

**Review by emiltin — COMMENTED** [review]
next time, please avoid changes not related to the PR, thinking about this file name change:
.cshareddata/xcschemes/I Bike CPH.xcscheme → ...eddata/xcschemes/I Bike CPH (da).xcscheme

**Comment by emiltin** [comment]
if you need to include changes from the develop branch into a feature branch, please rebase your feature branch on develop, instead of merginging develop into your feature branch

**Comment by emiltin** [comment]
please don't add a commit that just corrects a previous commit, like here:

baf3918 Follow-up on the previous commit: Use Danish labels instead of Englush
b7bd53a Fix to an issue where localisation key was shown instead of text

instead, merge the second commit into the first, either by using git commit --amend when adding the fix, or by squashing the two commits later using git rebase -i

**Comment by emiltin** [comment]
there are 3 commits in develop that are not in this branch, i assume this is because you've added them later?

**Comment by emiltin** [comment]
i have a hard time merging this cleanly. it's hard to follow  

can you please:
- rebase (rewrite) this branch on develop and squash into 1-5 meaningful commits
- check that it works
- force push the branch to github so i can review it again

**Comment by babak-feiz** [comment]
Sure

**Comment by babak-feiz** [comment]
I have created a new pull request: https://github.com/ibikecph/ibikecph-ios/pull/39
Let's follow up there.
